controllers/profiles/reconciler_dev.go

```diff
@@ -51,12 +56,46 @@ const (
 	recoverDeploymentErrorRetries = 3
 	// recoverDeploymentErrorInterval interval between recovering from failures
 	recoverDeploymentErrorInterval = 5 * time.Minute
+	extResPrefix                   = "sw.kogito.kie.org/resource-"
+	extResCamelSuffix              = "resource-camel"
+	extResGenericSuffix            = "resource-generic"
+	extResOpenAPISuffix            = "resource-openapi"
+	extResAsyncAPISuffix           = "resource-asyncapi"
 )
 
 type developmentProfile struct {
 	baseReconciler
 }
 
+type ConfigPath int
+
+const (
+	QUARKUS ConfigPath = iota
+	CAMEL
+	ASYNCAPI
+	OPENAPI
+	WORKFLOW
+	GENERIC
+)
+
+func getConfigMountPath(s ConfigPath) string {
+	switch s {
+	case QUARKUS:
+		return quarkusDevConfigMountPath
+	case CAMEL:
+		return quarkusDevConfigMountPath + "/routes"
+	case ASYNCAPI:
+		return quarkusDevConfigMountPath + "/templates"
+	case OPENAPI:
+		return quarkusDevConfigMountPath + "/specs"
+	case WORKFLOW:
+		return quarkusDevConfigMountPath + "/workflows"
+	case GENERIC:
+		return quarkusDevConfigMountPath + "/generic"
+	}
+	return quarkusDevConfigMountPath
+}
```

Comment:
Instead, you can have a `map[ConfigPath]string` variable. So your code would access this hash table in runtime instead of these conditionals.

Reply:
ok, going for the map